db/seeds.rb

```diff
@@ -42,8 +42,8 @@
     responseobj.update_attributes!(
       open_from: 1.minute.ago,
       invited_state: Response::SENT_STATE)
-    responseobj.initialize_invitation_token!
-    puts "mentor dagboek: #{Rails.application.routes.url_helpers.root_url}?q=#{responseobj.invitation_token.token}"
+    token = responseobj.initialize_invitation_token!
+    puts "mentor dagboek: #{Rails.application.routes.url_helpers.root_url}?q=#{person.external_identifier}#{token}"
```

Comment:
Ben ik nou gek? In routes.rb staat toch dat de / route en de questionnaire create route beide naar questionnaire#create gaan, en dat de ene een uuid (en dus niet een q) als parameter verwacht, dan verwacht de andere dat toch ook? Ik snap het niet meer denk ik.

Reply:
ja gek ben je sowieso, maar los daarvan:

Er gaat een `POST` route naar /, en ook een get. De get verwacht een token, de post een questionnaire answer. De `uuid` route is een parameter van `/questionnaire/`:

```ruby
  post '/' => "questionnaire#create"
  root to: 'token_authentication#show'
  resources :questionnaire, only: [:index, :show, :create], param: :uuid
```